srcpkgs/goldendict-ng/template

```diff
@@ -0,0 +1,22 @@
+# Template file for 'goldendict-ng'
+pkgname=goldendict-ng
+version=24.05.05
+revision=1
+_version_name=LiXia.ecd1138c
```

Comment:
```suggestion
archs="x86_64* aarch64*"
_version_name=LiXia.ecd1138c
```
Other arches are out due to `qt6-webengine` dependency.

Reply:
this does not need to be explicit, xbps-src will prevent build due to not being able to build qt6-webengine

Reply:
Really? AFAIK either `archs` or `broken` is needed, otherwise CI will fail with `failed to install target dependencies!`, as seen in the `i686` build in this PR. I don't think `broken` should propagate, so I went with `arch`s.

Reply:
well, even if it doesn't work, an arch restriction is the wrong solution, as it's not inherent to architectures but instead host/target wordsize:

https://github.com/void-linux/void-packages/blob/48ef82c9046ec34aa354bc669d919d52e50dd507/srcpkgs/kontact/template#L22-L24

Reply:
OK, so the sentiment is that `broken` *should* propagate. Let's go with that then.